spec/services/sitemap_indexer_spec.rb

```diff
@@ -14,13 +14,17 @@
       </urlset>
     SITEMAP
   end
-  let(:indexer) { described_class.new(sitemap_url: sitemap_url, domain: 'agency.gov') }
+  let(:indexer) do
+    described_class.new(
+      sitemap_url: sitemap_url,
+      domain: searchgov_domain.domain
+    )
+  end
 
   before do
     stub_request(:get, sitemap_url).
       with(headers: { 'User-Agent' => DEFAULT_USER_AGENT }).
       to_return(body: sitemap_content)
-    allow(searchgov_domain).to receive_message_chain(:reload, :index_urls)
```

Comment:
This line wasn't doing anything as written. The `index_urls` call is tested via:
```ruby
      it 'kicks off indexing' do
        allow(SearchgovDomain).to receive(:find_by).
          with(domain: 'agency.gov').and_return(searchgov_domain)
        expect(searchgov_domain).to receive_message_chain(:reload, :index_urls)
        index
      end
  ```